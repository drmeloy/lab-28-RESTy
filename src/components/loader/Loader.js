import React from 'react';
import PropTypes from 'prop-types';
import loader from '../../assets/loader.gif';
import styles from './Loading.css';

const Loader = ({ loading }) => {
  return loading && (
    <img className={styles.Loader} src={loader} alt='loader'></img>
  );
};

Loader.propTypes = {
  loading: PropTypes.bool
};

export default Loader;
