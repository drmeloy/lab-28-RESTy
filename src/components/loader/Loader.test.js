import React from 'react';
import { shallow } from 'enzyme';
import Loader from './Loader';

describe('Loader component', () => {
  it('renders Loader', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
