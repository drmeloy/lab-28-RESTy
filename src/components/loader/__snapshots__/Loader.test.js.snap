// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component renders Loader 1`] = `""`;
